spec/controllers/profile_controller_spec.rb

```diff
@@ -22,7 +24,66 @@
         expect(response.status).to eq(200)
       end
     end
-    
+
+    context 'changing passwords' do
+      it 'errors when new passwords dont match' do
+        @request.env["devise.mapping"] = Devise.mappings[:user]
+        @user = FactoryBot.create(:user_manager_user)
+        allow(request.env['warden']).to receive(:authenticate!).and_return(@user)
+        allow(controller).to receive(:current_user).and_return(@user)
+        sign_in @user
```

Comment:
these 5 lines already have a helper method `usermgr_stub_auth` in `/spec/support/auth_support.rb`